test/fixtures/util.cpp

```diff
@@ -108,12 +108,20 @@ uint64_t crc64(const std::string& str) {
     return crc64(str.data(), str.size());
 }
 
-PremultipliedImage render(Map& map) {
+PremultipliedImage render(Map& map, std::chrono::milliseconds timeout) {
```

Comment:
What's this used for? I'm suspicious of it because it looks like it has the potential to introduce timing-dependent flakiness to tests.